### PR TITLE
add A30M/A30T/Mizar/Mizar_S profiles

### PR DESCRIPTION
**Pull request by alexwoo1900**
Add profiles for 4 Geeetech machines

- A30M
- A30T
- Mizar
- Mizar S

### CODE REVIEW

**Comment by alexwoo1900** [comment]
Cura no longer supports third-party printers?

**Comment by nallath** [comment]
We do, we've just been very busy. I've merged a bunch of definitions yesterday.

File: resources/extruders/geeetech_A30M_extruder0.def.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "version": 2,
+    "name": "Extruder 1",
+    "inherits": "fdmextruder",
+    "metadata": {
+        "machine": "geeetech_A30M", 
+        "position": "0" 
+                      },
```
Comment (nallath) [inline resources/extruders/geeetech_A30M_extruder0.def.json:8]: This file has very strange formatting. Could you please pass it through a JSON linter?

File: resources/definitions/geeetech_A30M.def.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        "machine_height": {"default_value": 420},
+        "machine_depth": {"default_value": 320},        
+        "machine_center_is_zero": { "default_value": false},
+        "layer_height": { "default_value": 0.1 },
+        "layer_height_0": { "default_value": 0.3 },
+        "retraction_amount": { "default_value": 6 },
+        "retraction_enable": { "value": true },
+        "retraction_speed": { "default_value": 25 },
+        "adhesion_type": { "default_value": "skirt" },
+        "machine_head_with_fans_polygon": { "default_value": [[-75,35],[18,35],[18,-18],[-75,-18]] },
```
Comment (nallath) [inline resources/definitions/geeetech_A30M.def.json:26]:
```suggestion
        "machine_head_with_fans_polygon": { "default_value": [[-75, 35], [18, 35], [18, -18], [-75, -18]] },
```

File: resources/definitions/geeetech_A30T.def.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    "version": 2,
+    "name": "Geeetech A30T",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Teddy",
+        "manufacturer": "Geeetech",
+        "file_formats": "text/x-gcode",
+        "has_materials": true,
+        "machine_extruder_trains":  { "0": "geeetech_A30T_extruder0","1": "geeetech_A30T_extruder1","2": "geeetech_A30T_extruder2" }
```
Comment (nallath) [inline resources/definitions/geeetech_A30T.def.json:11]:
```suggestion
        "machine_extruder_trains":  { "0": "geeetech_A30T_extruder0", "1": "geeetech_A30T_extruder1", "2": "geeetech_A30T_extruder2" }
```

File: resources/definitions/geeetech_A30T.def.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        "machine_height": {"default_value": 420},
+        "machine_depth": {"default_value": 320},        
+        "machine_center_is_zero": { "default_value": false},
+        "layer_height": { "default_value": 0.1 },
+        "layer_height_0": { "default_value": 0.3 },
+        "retraction_amount": { "default_value": 6 },
+        "retraction_enable": { "value": true },
+        "retraction_speed": { "default_value": 25 },
+        "adhesion_type": { "default_value": "skirt" },
+        "machine_head_with_fans_polygon": { "default_value": [[-75,35],[18,35],[18,-18],[-75,-18]] },
```
Comment (nallath) [inline resources/definitions/geeetech_A30T.def.json:26]:
```suggestion
        "machine_head_with_fans_polygon": { "default_value": [[-75, 35], [18, 35], [18, -18], [-75, -18]] },
```

File: resources/definitions/geeetech_A30T.def.json [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+        "has_materials": true,
+        "machine_extruder_trains":  { "0": "geeetech_A30T_extruder0","1": "geeetech_A30T_extruder1","2": "geeetech_A30T_extruder2" }
+         },
+
+    "overrides": {
+        "machine_name": { "default_value": "Geeetech A30T" },
+        "machine_width": { "default_value": 320},
+        "machine_height": {"default_value": 420},
+        "machine_depth": {"default_value": 320},        
+        "machine_center_is_zero": { "default_value": false},
```
Comment (nallath) [inline resources/definitions/geeetech_A30T.def.json:19]:
```suggestion
```

DRY principle. This is already the default, no need to redefine it.

**Review by nallath — CHANGES_REQUESTED** [review]
There also seems to be a merge conflict. It seems that someone else has already contributed one of the machines that you're trying to add.

**Pull request closed without merging**